Snakemake/workflow/rules/ancestral_inference.smk

```diff
@@ -69,33 +69,50 @@ if config['ancestralAllele'] == None:
             vcfAlleles=rules.extract_snps_from_info.output
         conda: "HLab_tsinfer"
         output:
-            expand("../Project/AncestralAllele/Estsfs/EstSfs_Dict{{chunk}}.csv")
+           [f"../{Project}/AncestralAllele/Estsfs/" + x for x in expand("EstSfs_Dict{{chunk}}.csv", chunk = range(config['no_estsfs_chunks']))]
         wildcard_constraints:
             chunk="\d+"
         params:
-            outDir="../Project/AncestralAllele/Estsfs",
-            noCycle=config['noEstSfsChunks']
-        threads: config['noEstSfsChunks']
+            outDir=f"../{Project}/AncestralAllele/Estsfs",
+            noCycle=config['no_estsfs_chunks']
+        threads: config['no_estsfs_chunks']
         resources: cpus=1, mem_mb=64000, time_min=360
-        log: 'logs/CreateEstsfsDicts{chunk}.log'
+        log: 'logs/Create_estsfs_dicts{chunk}.log'
         shell:
-            "python scripts/CreateInputForEstsfs_fromWGAbed_Cactus.py {wildcards.chunk} {params.noCycle} {input.alignedAlleles} {input.vcfAlleles} {params.outDir}"
+            "python scripts/Create_estsfs_input_fromWGAbed_Cactus.py {wildcards.chunk} {params.noCycle} {input.alignedAlleles} {input.vcfAlleles} {params.outDir}"
             #CreateInputForEstsfs_Loop.sh This is qsub
 
+    rule extract_estsfs_pos:
+        input:
+            rules.create_estsfs_dicts.output
```

Comment:
This rule was added because we didn't track the positions of the SNPs that went into the inference